NEW_README.md

```diff
@@ -188,29 +188,43 @@ bundle exec rake opsworks_performance    # (takes about 1.5 hours)
 
 ## Soak performance tests
 
-In order to execute a Soak performance run:
+The soak performance test executes a long-running scenario under medium load:
+* 14 days
+* 600 agents
+* `role::by_size::large`
 
-* Follow the instructions from
-[Just set up PE and Gatling](#set-up-puppet-enterprise-and-gatling-but-do-not-execute-a-gatling-scenario).
+### Provision, tune, run
 
-* Then manually tune the master by running `puppet infrastructure tune` on the master.
+A set of 'soak' rake tasks are provided to handle setup and test execution, allowing nodes to be provisioned as part of the run or as a separate step.
+The pre-suite includes tuning of the master via 'puppet infrastructure tune' so this is no longer a manual step.
```

Comment:
person reading this likely doesn't have history, so the statement "so this is no longer a manual step" can go.

Reply:
Updated